pkg/asset/installconfig/vsphere/validation.go

```diff
@@ -66,3 +81,78 @@ func folderExists(ic *types.InstallConfig, fldPath *field.Path) field.ErrorList
 	}
 	return nil
 }
+
+func validatevCenterVersion(vim25 *vim25.Client) error {
+	constraints, err := version.NewConstraint("<= 6.7")
+
+	if err != nil {
+		return err
+	}
+
+	vCenterVersion, err := version.NewVersion(vim25.ServiceContent.About.Version)
+	if err != nil {
+		return err
+	}
+
+	if constraints.Check(vCenterVersion) {
```

Comment:
TODO: The version is checked but the exact build of U3 is not. Add that check as well so that we don't notify when a vSphere environment is up-to-date.